### ISSUE TITLE
aspcud not working with 22.11

### ISSUE THREAD

**Issue by hugosenari** [NixOS/nixpkgs#206333]
### Describe the bug
`aspcud` uses `clasp` and `gringo` (or `clingo` ?)
For this reason, I can't say if this issue is with aspcud.

But when I run it, it throws the following error:
> ERROR: (clasp): Parse Error: Line 1, 'a': Unrecognized input format!

### Steps To Reproduce
I created one example:

https://github.com/riinr/flake-crown/blob/flake-pinning/pkgsRev/example/flake.nix#L14
It has 3 outputs:

1. __default__ with nixpkgs 22.05 (works)
   -  `nix build --refresh 'github:riinr/flake-crown?dir=pkgsRev/example'#default`
2. __nixpkgsMast__ with master  (fail)
   -  ` nix build --refresh 'github:riinr/flake-crown?dir=pkgsRev/example'#nixpkgsMast`
3. __nixpkgs2211__ withs 22.11  (fail)
   -  `nix build --refresh 'github:riinr/flake-crown?dir=pkgsRev/example'#nixpkgs2211`

### Expected behavior
It should work like works like previous versions (or not?)

### Additional context
22.05:
- aspcud-1.9.5
- gringo-4.5.4
- clasp-3.1.4

22.11:
- aspcud-1.9.6
- clingo-5.6.2
- clasp-3.1.4

### Notify maintainers
- @vbgl 
- @dotlambda 
- @hakuch

### Metadata

  /nix/store/jkv35nzx1nh8g0d5p94k8i4zjg8j8i37-nix-info
copying path '/nix/store/jkv35nzx1nh8g0d5p94k8i4zjg8j8i37-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.79, NixOS, 23.05 (Stoat), 23.05.20221122.2cb8f1a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(hugosenari): `"nixpkgs-22.05, unstable"`
 - channels(root): `"nixos, unstable"`
 - nixpkgs: `/home/hugosenari/.nix-defexpr/channels/nixpkgs`

**Comment by vbgl** [comment]
Fixed by #208081?

**Comment by hugosenari** [comment]
Wasn't working, then I realized that I was using the wrong URI in my inputs.

```nix
{
  inputs.unstable.url = "github:NixOS/nixpkgs";                 fixed
  inputs.r2211.url    = "github:NixOS/nixpkgs/release-22.11";   backported
  # wrong URI
  inputs.t2211.url    = "github:NixOS/nixpkgs/22.11";           not backported
  inputs.n2211.url    = "github:NixOS/nixpkgs/nixos-22.11";     not backported
}
```

Thank you :confetti_ball:
